modules/@apostrophecms/area/ui/apos/components/AposAreaMenu.vue

```diff
@@ -4,7 +4,7 @@
       :button="buttonOptions"
       v-bind="extendedContextMenuOptions"
       @open="menuOpen"
-      @close="menuClose"
```

Comment:
We're not using the close event here at all?

Reply:
you are correct this is still in play, re implemented